src/open_company_web/core.cljs

```diff
@@ -92,6 +93,9 @@
       #(handle-url-change %))
     (.setEnabled true)))
 
+;; setup Sentry error reporting
+(raven-setup)
```

Comment:
Why put this down here... rather than at the top of the file. Won't we miss problems that happen in this file above this line?